api/idea-card/models/idea-card.settings.json

```diff
@@ -19,23 +19,15 @@
     },
     "tagline": {
       "type": "text",
-      "required": true
+      "required": false
     },
     "description": {
       "type": "text",
       "required": true
     },
     "targetAudience": {
       "type": "text",
-      "required": true
-    },
-    "hourCommitmentMin": {
```

Comment:
@dbradham can you double check this is not used anymore?

Reply:
I can confirm this.